POM/loginPage_PO.js

```diff
@@ -0,0 +1,44 @@
+class Login {
+    typeUserName(username) 
+    {
+      cy.get("#Username").clear().type(username);
+      return this;
+    }
+  
+    typePassword(password) {
+      cy.get("#Password").clear().type(password);
+      return this;
+    }
+  
+    clickSubmit() {
+      cy.get("input[name='login']").click();
+      return this;
+    }
+    
+    clickClear() {
+      cy.get("input[name='clear']").click();
+      return this;
+    }
+  
+    verifyHeading() {
+      cy.get("h3").should("have.text", "Welcome elisa");
```

Comment:
Do we need this method? I don't think this method is  used anywhere. Please review your code yourself once before sending to PR